src/main/java/com/sap/sgs/phosphor/fosstars/model/feature/oss/OssFeatures.java

```diff
@@ -216,4 +219,19 @@ private OssFeatures() {
    */
   public static final Feature<Boolean> SIGNS_ARTIFACTS
       = new BooleanFeature("If a project signs artifacts");
+
+  /**
+   * If OWASP Dependency plugin is used to scan the project. It is either used as a mandatory step,
```

Comment:
The tools is called OWASP Dependency Check. It's not required to use it via plugins. In fact, there are multipel plugins for the tool. Please update the description.
```suggestion
   * Shows if OWASP Dependency Check is used to scan a project. It is either used as a mandatory step,
```